std/internal/math/biguintcore.d

```diff
@@ -52,6 +52,130 @@ import std.range.primitives;
 import std.traits;
 
 private:
+
+// compile-time & run-time dipatchers to allows BigInt CTFE for 32 bit systems.
```

Comment:
typo: "to allow:

Also as I mentioned earlier, the dispatchers are more general than 32 bit, so either X86 or remove the last part of the sentence.

Reply:
Should be much better now, in no way someone will ever be confused about the role of the new functions.